polyfills/Array/prototype/findIndex/config.toml

```
@@ -15,16 +15,16 @@ docs = "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global
 [browsers]
 bb = "6 - 10"
 chrome = "<=45"
```

Looks like Chrome 45 supports `Array.findIndex`?

```suggestion
 chrome = "<45"
```